### DISCUSSION TITLE
Do commits that were authored by me button committed by me count as a contribution?

### DISCUSSION THREAD

**Discussion by logankilpatrick** [community/community#21773] [New to GitHub]
<p>Hey all, from what I understand, the graph on my profile shows commits that were committed by me meaning if I author a commit and it’s committed by someone else, I will not get commit activity for this. Am I correct here? </p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/logankilpatrick">@logankilpatrick</a>!</p>
<p>Great observation! Yes, the contribution qualification is based on who committed the code into the repository <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile" rel="nofollow noopener">according to the criteria mentioned here</a>.  Additionally, I am not sure if you are familiar with the “<a href="https://github.community/t/getting-all-your-commits-in-your-contributions-graph/10186">Getting all your commits in your contributions graph</a>” article, but there is mention of a fun herokuapp, toward the end, that helps with finding whether a specific commit to a public repository meets all the criteria for making your graph.  Let me know if that helps.</p>
